### PR TITLE
changed include and exclude file option to be accepted multiple times.

### PR DESCRIPTION
**Pull request by prozessorkern**
Attempt to address #26 to include/exclude multiple patern.

### CODE REVIEW

File: metrixpp/ext/std/tools/collect.py [hunk 1]
```diff
@@ -46,14 +46,23 @@ def configure(self, options):
...
-        try:
-            self.add_include_rule(re.compile(options.__dict__['include_files']))
-        except Exception as e:
-            self.optparser.error("option --include-files: " + str(e))
-        try:
-            self.add_exclude_rule(re.compile(options.__dict__['exclude_files']))
-        except Exception as e:
-            self.optparser.error("option --exclude-files: " + str(e))
+        if options.__dict__['include_files']:
+            print(options.__dict__['include_files'])
```
Comment (avkonst) [inline metrixpp/ext/std/tools/collect.py:50]: forgotten debugging line?

File: metrixpp/ext/std/tools/collect.py [hunk 2]
```diff
@@ -46,14 +46,23 @@ def configure(self, options):
...
+            try:
+                for include_rule in options.__dict__['include_files']:
```
Comment (avkonst) [inline metrixpp/ext/std/tools/collect.py:52]: how does it know if the option is given once or many times? Will it be always an array?

File: metrixpp/ext/std/tools/collect.py [hunk 3]
```diff
@@ -46,14 +46,23 @@ def configure(self, options):
...
+                    self.add_include_rule(re.compile(include_rule))
+            except Exception as e:
+                self.optparser.error("option --include-files: " + str(e))
+        else:
+            self.add_include_rule(re.compile(r'.*'))
+        if options.__dict__['exclude_files']:
+            try:
+                for exclude_rule in options.__dict__['exclude_files']:
```
Comment (avkonst) [inline metrixpp/ext/std/tools/collect.py:60]: same question here

File: metrixpp/ext/std/tools/collect.py (hunk 1 above)
Comment (prozessorkern) [reply metrixpp/ext/std/tools/collect.py]: Thanks - removed the line

File: metrixpp/ext/std/tools/collect.py (hunk 2 above)
Comment (prozessorkern) [reply metrixpp/ext/std/tools/collect.py:52]: 'include_files' is None if no rule is given and contains a list of rules if one ore more rules are passed.

So if there is no rule passed by the user include_files are None and the else path is taken which applies the default rule.

File: metrixpp/ext/std/tools/collect.py (hunk 3 above)
Comment (prozessorkern) [reply metrixpp/ext/std/tools/collect.py:62]: see conversation at include_files

**Pull request merged**